public/javascripts/playlist/playlistTable.js

```diff
@@ -36,22 +38,37 @@ let playlistTable = {
                   pl.name
                 )
               ]),
-              m('td', [
+              m(
+                'td',
                 m(
-                  'button.btn.btn-default.pull-right',
-                  {
-                    onclick: function() {
-                      if (confirm('Do you want to delete this playlist?')) {
-                        deletePlaylist(pl.name);
-                        remove(args.currentPlaylists(), function(n) {
-                          return n.name === pl.name;
-                        });
-                      }
-                    }
-                  },
-                  'Delete Playlist'
+                  '.row',
+                  m('.col-xs-12', [
+                    m(
+                      'button.btn.btn-default.pull-right',
+                      {
+                        onclick: () => {
+                          if (confirm('Do you want to delete this playlist?')) {
+                            deletePlaylist(pl.name);
```

Comment:
It should delete by id, not name, because if it is by name, the '#' symbol will still cause issues.

Reply:
I will change it.